staging/src/k8s.io/apiserver/pkg/util/flowcontrol/max_seats_test.go

```diff
@@ -130,7 +131,7 @@ func Test_GetMaxSeats(t *testing.T) {
 					},
 				},
 			}
-			if _, err := c.digestConfigObjects([]*flowcontrolv1.PriorityLevelConfiguration{testPriorityLevel}, nil); err != nil {
+			if _, err := c.digestConfigObjects(context.Background(), []*flowcontrolv1.PriorityLevelConfiguration{testPriorityLevel}, nil); err != nil {
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.

Reply:
done